md/migrate-from-an-earlier-version-of-bonita-bpm.md

```diff
@@ -298,13 +298,35 @@ please follow this procedure:
 
 ## Migrating to Bonita 7.9+ using Oracle
 
-Bonita 7.9+ supports Oracle 12c (12.2.x.y) version. To migrate to Bonita 7.9+ when using Oracle,
-please follow this procedure:
-* ensure your Bonita platform is shut down
-* in a first step, run Bonita migration tool to update Bonita platform to version 7.8.4, following the procedure above
-* then upgrade your Oracle database server to the version 12c (it must be 12.2.x.y)
-* in a second step, run the migration tool again to migrate the platform to 7.9.0 or newer
-* once done, you can restart your updated Bonita platform
+Bonita 7.9+ supports Oracle 12c (12.2.x.y) version: this is a requirement change.
+
+The Oracle database server change needs to be done before using the Bonita migration tool from 7.8.4 to 7.9.0.
+
+### Migrate to 7.8.4
+
+Skip this section and jump directly to **Upgrade Oracle database server** section if the 7.8.4 is already the version in use.
+
+* shut down the Bonita platform 
+* run Bonita migration tool to update Bonita platform to version 7.8.4, following the procedure above
+
+### Upgrade Oracle database server
+
+* shut down the Bonita platform
+* upgrade the Oracle database server to the version 12c (it must be 12.2.x.y)
+
+### Configure the Oracle database server
+
+* configure the Oracle database server, in particular activate the XA transactions management: see the *Oracle Database* section in the [Database creation and configuration for Bonita engine and BDM](database-configuration) page:
+* Install the missing Oracle components
+* execute the SQL scripts to *install* XA management elements
+* execute the SQL requests to GRANT the proper rights to the Oracle user
+
+### Migrate to 7.9+
+
+* shut down the Bonita platform 
+* Run the migration tool to migrate the platform to 7.9+, following the procedure above
+* install target Tomcat application server with bonita 
```

Comment:
... Is it necessarily a Tomcat, in 7.9+?
I believe we could still have Wildfly in 7.9.x (however deprecated it may have been)...